ui/src/main/resources/edu/wpi/grip/ui/templates/operations/Switch.vm

```diff
@@ -0,0 +1,8 @@
+    public static void Switch(boolean sw, Object onTrue,Object onFalse, Object output){
```

Comment:
This method won't actually do anything. You're just reassigning a local reference.

Why not something like

```
public static <T> T Switch(boolean sw, T onTrue, T onFalse) {
    return sw ? onTrue : onFalse;
}
```